lib/Components/side_drawer.dart

```diff
@@ -70,97 +75,115 @@ class _SideDrawerState extends State<SideDrawer> {
                 ),
               ],
             ),
-            ModulesCard(cardLine: 'DashBoard', pageMover: '/dashboard'),
-            Card(
-              color: Colors.black,
-              child: GestureDetector(
-                //behaviour to translucent to get Tap even on blank or empty space within container
-                behavior: HitTestBehavior.translucent,
-                onTap: () {
-                  count++;
-                  setState(() {
-                    count % 2 == 0 ? _loading = false : _loading = true;
-                  });
-                },
-                child: Padding(
-                  padding: const EdgeInsets.all(16.0),
-                  child: Row(
-                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                    children: [
-                      Text(
-                        'Modules',
-                        style: TextStyle(
-                          fontSize: 16.0,
-                          color: Colors.white,
-                        ),
-                      ),
-                      Icon(
-                        Icons.arrow_downward,
-                        color: Colors.deepOrangeAccent,
-                      ),
-                    ],
-                  ),
-                ),
-              ),
-            ),
-            _loading
-                ? Card(
-                    color: Colors.black,
-                    child: Column(
-                      crossAxisAlignment: CrossAxisAlignment.center,
+            ModulesCard(cardLine: 'Dashboard', pageMover: '/dashboard'),
```

Comment:
@TusharGupta03 can you confirm if these changes are valid. I thought side_drawer was removed from components and we were only considering side_drawer2 only.

Reply:
We made these changes earlier when the new side drawer was not implemented. We are not using this anywhere now.